google/cloud/bigquery_storage_v1/types/stream.py

```diff
@@ -194,6 +194,8 @@ class WriteStream(proto.Message):
             generate data that's compatible with this schema to send in
             initial ``AppendRowsRequest``. The table schema could go out
             of date during the life time of the stream.
+        write_mode (google.cloud.bigquery_storage_v1.types.WriteStream.WriteMode):
+            Immutable. Mode of the stream.
```

Comment:
Not a terribly useful comment, nor does the `WriteMode` enum provide any help on the semantics.